src/App.jsx

```diff
@@ -80,545 +67,446 @@ class App extends Component {
                                 <p>A keep is an off-chain container for private data. Keeps help contracts harness the full power of the public blockchain &mdash; enabling deep interactivity with private data.</p>
                             </Col>
+                            <Col xs={5} className="col-circles">
+                                <Picture src={getSrc('/images/texture-circle', 'png', 3)} />
+                            </Col>
                         </Row>
+                    </PageSection>
+                    <PageSection id={sections.SLACK}>
                         <Row className="slack-signup">
-                            <Col sm={12} md={2} mdPush={3} className="col-icon">
-                                <img alt="Slack Logo" src={Icons.slackGreen} />
-                            </Col>
-                            <Col sm={12} md={4} mdPush={3}>
+                            <Col xs={7}>
                                 <EmailForm
                                     label="Slack Email"
-                                    btnText="join"
                                     onSubmit={signupSlack}
                                     requestStates={ajaxRequestStates}
-                                    request={actionTypes.SIGNUP_SLACK} />
+                                    request={actionTypes.SIGNUP_SLACK}>
+                                    <h3>
+                                        Join our community on
+                                        <Icons.Slack color="#AC6E16"/>
+                                    </h3>
+                                </EmailForm>
+                            </Col>
+                            <Col xs={5} className="col-circles">
+                                <div>
+                                    <Picture src={getSrc('/images/texture-circle-2', 'png', 3)} />
+                                </div>
                             </Col>
                         </Row>
                     </PageSection>
-                    <PageSection id={sections.ABOUT}>
+                    <PageSection id={sections.GITHUB}>
                         <Row>
-                            <Col sm={12} md={6} className="mailing-list">
-                                <img alt="Email" src={Icons.email} />
-                                <h3>Join the Mailing List</h3>
-                                <p>Stay informed</p>
-                                <EmailForm
-                                    label="Email"
-                                    btnText="join"
-                                    onSubmit={signupMailingList}
-                                    requestStates={ajaxRequestStates}
-                                    request={actionTypes.SIGNUP_MAILING_LIST} />
+                            <Col xs={12} md={12}>
+                                <h3>
+                                    View the <Icons.Github /> Repository
```

Comment:
Any chance we can make GitHub proper text here? Seems like it might just be the font with particular weight + color combo?

Reply:
I went with the last option.

Reply:
Ah, makes sense! This still copy/pastes weird, but should be good enough for e.g. screen readers.